src/internal/nativeInterface.ts

```diff
@@ -32,9 +32,11 @@ export default {
   ...RNCNetInfo,
   get eventEmitter(): NativeEventEmitter {
     if (!nativeEventEmitter) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       /// @ts-ignore
       nativeEventEmitter = new NativeEventEmitter(RNCNetInfo);
```

Comment:
This one actually worries me a little bit. I know the NativeEventEmitter interfaces / functionality changed recently (I had to handle them here https://github.com/invertase/react-native-firebase/commit/262452d69c2dadd79475235fca42c12b18b2e208#diff-82b084457a12d227f6ce7e525327c3c094a80001c1f4e987086fb7ca0c582899) - I think at some point we need to remove the ts-ignore, and actually make sure it works :worried:

Reply:
if remove `@ts-ignore` we will gets error

![image](https://user-images.githubusercontent.com/51177449/135210830-dd983c69-196d-4226-b898-26a94005342a.png)


I don't have enough time to fix this:c

+ I think, this is another problem not related to jcenter